feat(index.js): added labels for markdown and preview

arteevraina opened the pull request:
##  Fixes Issue #345  #337 
Separate labels for `Markdown` and `Preview` have been added.

## Screenshot
![image](https://user-images.githubusercontent.com/43968121/79763163-51fefe00-8341-11ea-9be0-d372cd51c2f5.png)


Please Review and suggest changes @irmerk @Michael-Grover @DianaLease  :)

DianaLease commented on examples/src/index.js:
```diff
@@ -140,7 +142,7 @@ function Demo() {
           </Grid.Column>
         </Grid>
-        <Divider vertical>Preview</Divider>
+        <Divider vertical></Divider>
```
This can be self-closing (`<Divider vertical />`

arteevraina replied:
Yes, made the change. Thanks :+1:

irmerk commented:
I want to wait for #315 to be finished to see how that would affect this PR.

arteevraina commented:
## With Shadow
![image](https://user-images.githubusercontent.com/43968121/79799781-a96c9080-8378-11ea-8612-aacba7bddb18.png)
## Without Shawdow
![image](https://user-images.githubusercontent.com/43968121/79799833-c4d79b80-8378-11ea-84a6-bcdc86d72d25.png)

@irmerk @DianaLease @Michael-Grover I have added the shadow under formatting toolbar and changed the label size to 18px.

irmerk commented:
You'll need to rebase. to resolve conflicts.

arteevraina commented:
@irmerk Do I still need to `rebase` it if it does not have merge conflicts?

irmerk commented:
Yeah there are still conflicts, and we don't want merge commits in the git log, so you'll need to interactive rebase and get rid of that.

arteevraina commented:
Sure @irmerk I will do that :)

arteevraina commented:
@irmerk I will close this PR and create a fresh PR for this.
